includes/class-sensei-question.php

```diff
@@ -1126,7 +1126,7 @@ public static function file_upload_load_question_data( $question_data, $question
 			$answer_media_filename = '';
 
 			$question_helptext = '';
-			if ( isset( $question_data['question_wrong_answers'][0] ) ) {
+			if ( is_array( $question_data['question_wrong_answers'] ) && isset( $question_data['question_wrong_answers'][0] ) ) {
```

Comment:
@merkushin  Note that this check was not here before, realized that in other function we were accessing $question_data['question_wrong_answers'] first element, before check-ing if it is an array

Reply:
Nice catch!